### ISSUE TITLE
Minor startup issue.

### ISSUE THREAD

**Issue by rquadling** [Nilhcem/FakeSMTP#31]
Hi.

When I load FakeSMTP via ...

```java -jar fakeSMTP-2.0.jar -a 127.0.0.1 -p 25 -s -b -o D:\SMTP```` (yes I'm on Windows).

I get ...

```
27 May 2015 11:19:12 ERROR com.nilhcem.fakesmtp.core.I18n - Can't find bundle for base name i18n/messages, locale en_GB
27 May 2015 11:19:12 INFO  com.nilhcem.fakesmtp.core.I18n - Will use default bundle (en_US) instead
27 May 2015 11:19:12 INFO  org.subethamail.smtp.server.SMTPServer - SMTP server /127.0.0.1:25 starting
27 May 2015 11:19:12 INFO  org.subethamail.smtp.server.ServerThread - SMTP server /127.0.0.1:25 started
```

With regard to the locale messages, is there anything important to worry about?

If not, then just close this issue.

Regards,

Richard Quadling.

**Comment by Vest** [comment]
Just a small tip. To reproduce the issue, I had to change the user.language:
```
java -Duser.language=gb -jar ...
```

**Comment by rquadling** [comment]
Just copy the current *US* to *GB*. There isn't anything I would say/write differently.

**Comment by Vest** [comment]
I have checked the US resource bundle, and haven't found words that are specific for GB. Probably instead of duplicating resources, maybe it would make sense to "generalize" some of them.

**Comment by rquadling** [comment]
If that's possible. I'm not a Java developer, so I don't know how to abstract that. But I feel generalizing them would be overkill. Unless the intent is to support that for other shared languages (Spanish, Portuguese, etc.) But being only an English speaker, I'm not sure how wide spread such generalisation would be to warrant the additional code.

**Comment by Vest** [comment]
@rquadling, I have renamed the resource file for English only, because I didn't find strings that could be different in US and GB. The result is in my branch: [Fix/Issue31] (https://github.com/Vest/FakeSMTP/tree/Fix/Issue31). If you have time, could you please verify whether the warning disappears or not?

I do not suggest to do this for Spanish and/or Portuguese (or other languages), because they might have significant difference. To be honest, we have a pt_BR resource only :)

@Nilhcem, I apologize for bothering you, could you please take a look at this issue as well? Was my proposal acceptable, or you have a better one?

**Comment by rquadling** [comment]
@Vest Sorry, I'm only a user, not a java dev.

**Comment by Vest** [comment]
Ok then, I thought you've build FakeSMTP before using it.
No worries.

**Comment by Nilhcem** [comment]
@rquadling,  @Vest,
Thanks, I finally removed the i18n country specifier for every i18n bundles to keep only the language,
Changes will be included when the next version (2.1) is published
